src/patterns/addresses/multiple/index.njk

```diff
@@ -18,16 +18,16 @@ stylesheets:
 
   {{ govukInput({
     label: {
-      html: 'Address line 1'
+      text: 'Address line 1'
     },
     id: "address-line-1",
     name: "address-line-1",
-    autocomplete: "address-line1"
+    autocomplete: "address-line-1"
```

Comment:
The `autocomplete` attribute requires [specific values](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) to work – it looks like `address-line1` is the correct / valid version, so this change should not be made.

```suggestion
    autocomplete: "address-line1"
```